Documentation/cross-building.md

```diff
@@ -0,0 +1,76 @@
+Cross Compilation for ARM on Linux
+==================================
+
+Through cross compilation, on Linux it is possible to build CoreCLR for arm or arm64.
+
+Requirements
+------------
+
+You need a Debian based host and the following packages needs to be installed:
+
+    $ sudo apt-get install qemu qemu-user-static binfmt-support debootstrap
+
+In addition, to cross compile CoreCLR the binutils for the target are required. So for arm you need:
+
```

Comment:
If my CoreCLR patch is merged then they will be the same apart from the second half of the document in CoreCLR which is to do the cross compilation on Windows.